site/content/docs/user/rootless.md

```diff
@@ -31,15 +32,17 @@ Delegate=yes
 ```
 iptable_nat
 ip6table_nat
+ip6_tables
 ```
 
 ## Restrictions
 
 The restrictions of Rootless Docker apply to kind clusters as well.
 
 e.g.
-- OverlayFS cannot be used unless the host is using kernel >= 5.11, or Ubuntu/Debian kernel
-- Cannot mount block storages
+- OverlayFS cannot be used unless the host is using kernel >= 5.11, or
+  Ubuntu/Debian kernel
```

Comment:
What is the purpose of splitting this line?
Unless we're going to require a particular auto formatter I'd rather not endlessly reformat lines like this arbitrarily. This doesn't seem to have any content change.

Reply:
https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md#line-length

Reply:
That style guide is specifically for that repo owned by SIG Contributor Experience and is not agreed upon by the rest of the project 🙃 

> This style guide is for content in the Kubernetes github community repository. It is an extension of the Kubernetes documentation style-guide.

I would really prefer to not have this re-formatted by competing settings. If any auto-formatting is to be used here it should be enforced by a linter in presubmit.

https://kubernetes.io/docs/contribute/style/style-guide/ is the guide used for kubernetes.io, which is a closer analogue of site/ here.